LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_AnalogInput.TcPOU

```diff
@@ -15,9 +15,24 @@ VAR_INPUT
     fTermMax: LREAL;
     // The fReal value correlated with the terminal's min value
     fTermMin: LREAL;
+    // Value to scale the end result to
+    {attribute 'pytmc' := '
+        pv: RES
```

Comment:
Not sure if we have a standard PV for this.

Reply:
I think RES is a good pick